javatests/arcs/core/storage/StorageProxyTest.kt

```diff
@@ -230,7 +230,7 @@ class StorageProxyTest {
         storageProxy: StorageProxy<CrdtData, CrdtOperationAtTime, String>,
         reader: Boolean
     ) = mock<Callbacks<CrdtOperationAtTime>>().let {
-        HandleWithCallback(Handle(name, storageProxy, it, reader), it)
+        HandleWithCallback(Handle(name, storageProxy, it, reader, true), it)
```

Comment:
shouldn't there be something in the tests for the non-writable vs writable handle? it's totally fine to add it later though.

Reply:
We can't enforce it as this level like TS does with the current design. This is just using Handle as a holder for this value, and subclasses that /do/ have write methods can check it if they want additional runtime checking.